tests/unit-tests/crud/data.php

```diff
@@ -139,7 +139,7 @@ function test_get_meta_data_shared_bug() {
 	 */
 	function test_get_meta_data_cache_invalidation() {
 		$object  = new WC_Order;
-		$object->add_meta_data( 'test_meta_key', 'val1', true );
+		$object->add_meta_data( 'test_meta_key', array( 'val1' ), true );
```

Comment:
It was a testing directly in the test unit, I could undo that and just add a new a method.